api/service/restclientsupport/service.go

```diff
@@ -0,0 +1,172 @@
+package restclientsupport
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	msg_pb "github.com/harmony-one/harmony/api/proto/message"
+	"github.com/harmony-one/harmony/internal/utils"
+)
+
+// Constants for rest client support service.
+const (
+	Port = "30000"
```

Comment:
another port?  we need a central place to keep track of all open ports.  I've assigned a design issue to you and @harmony-ek . https://github.com/harmony-one/harmony/issues/612